spec/mailers/subscription_spec.rb

```diff
@@ -38,11 +38,11 @@
     expect(body_lines[3]).to match(/#{I18n.t('subscriptions.email.confirmation.subheading', email: email)}/)
     expect(body_lines[5]).to match(/\* Subject: English/)
     expect(body_lines[6]).to match(/\Suitable for NQTs/)
-    expect(body_lines[8]).to include('You&#39;ll receive a single job alert email at the end of any day')
+    expect(body_lines[8]).to include('You&#39;ll receive')
```

Comment:
So we don't need to change tests when this text is corrected (we don't send jobs at the end of the day)